raiden/tests/integration/test_integration_events.py

```diff
@@ -419,13 +419,13 @@ def test_secret_revealed(raiden_chain, deposit, settle_timeout, token_addresses)
     settle_expiration = app0.raiden.chain.block_number() + settle_timeout
     wait_until_block(app0.raiden.chain, settle_expiration)
 
-    assert_synched_channel_state(
+    assert_synced_channel_state(
         token_address,
         app1, deposit - amount, [],
         app2, deposit + amount, [],
     )
 
-    assert_synched_channel_state(
+    assert_synced_channel_state(
```

Comment:
Lol :) `synch` versus `sync` is a long discussion we have had. I also prefer `sync` but apparently `synch` is also an acceptable abbreviation.